src/unix/core.c

```diff
@@ -1557,13 +1557,9 @@ int uv_gettimeofday(uv_timeval64_t* tv) {
 }
 
 void uv_sleep(unsigned int msec) {
-  unsigned int sec;
-  unsigned int usec;
-
-  sec = msec / 1000;
-  usec = (msec % 1000) * 1000;
-  if (sec > 0)
-    sleep(sec);
-  if (usec > 0)
-    usleep(usec);
+  struct timespec timeout;
+
+  timeout.tv_sec = msec / 1000;
+  timeout.tv_nsec = (msec % 1000) * 1000 * 1000;
+  nanosleep(&timeout, NULL);
```

Comment:
Does it make sense handling `EINTR` errors?

Reply:
@cjihrig That seems like a question for you. You probably had your reasons for making uv_sleep() public.

Reply:
I guess it makes sense to handle `EINTR` and resume sleeping, because I don't think the Windows implementation can be interrupted.

Reply:
Sounds good. PR updated and new CI kicked off, PTAL.